packages/compass-smoke-tests/src/cli.ts

```diff
@@ -104,7 +107,14 @@ const argv = yargs(hideBin(process.argv))
     type: 'boolean',
     description: 'Do not delete the sandbox after a run',
     default: false,
-  });
+  })
+  .option('tests', {
+    type: 'string',
+    choices: SUPPORTED_TESTS,
+    description: 'Which tests to run',
+  })
+  .array('tests')
+  .default('tests', []);
```

Comment:
I _think_ this is doable in a single call of the `option` builder function 🤔

```suggestion
  .option('tests', {
    type: 'array',
    string: true,
    choices: SUPPORTED_TESTS,
    description: 'Which tests to run',
    default: [],
  });
```

I think th